src/mono/wasm/runtime/pinvoke-tables-default-netcore.h

```diff
@@ -0,0 +1,229 @@
+// GENERATED FILE, DO NOT MODIFY
```

Comment:
Do we have to commit generated file?

Reply:
This file needs to be generated by a tool which is ran on the BCL assemblies. Not sure where this can be done during the build.

Reply:
It might be easier to run it after each native library build and include them as individual files.

@safern @ViktorHofer any recommendations?

Reply:
How is this exactly generated? Do you run the tool against all managed assemblies?

Reply:
This is now generated.